homeassistant/components/husqvarna_automower_ble/__init__.py

```diff
@@ -0,0 +1,67 @@
+"""The Husqvarna Autoconnect Bluetooth integration."""
+
+from __future__ import annotations
+
+import logging
+
+from automower_ble.mower import Mower
+from bleak_retry_connector import close_stale_connections_by_address, get_device
+
+from homeassistant.components import bluetooth
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_ADDRESS, CONF_CLIENT_ID, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import DeviceInfo
+
+from .const import DOMAIN, MANUFACTURER
+from .coordinator import Coordinator
+
+LOGGER = logging.getLogger(__name__)
+
+PLATFORMS = [
+    Platform.LAWN_MOWER,
```

Comment:
Should also include `Platform.SENSOR` and separate sensor entities to the sensor platform